src/components/ComponentPages/DirectSupportedCamps/__test__/directSupportedCamps.test.tsx

```diff
@@ -195,7 +195,7 @@ describe("Direct Support camps page", () => {
       />
     );
     expect(screen.getByText("Remove")).toBeTruthy();
```

Comment:
Please move "Remove" to common message or label file

Reply:
Done